Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM python:2.7-alpine
-RUN apk add --update git openssl
+RUN apk add --update openssl
```

Comment:
If we move the `ircNotification` to run outside of the Docker image (like we did for Mozillians) then we can probably remove `openssl` too. Perhaps a follow-up?

Reply:
Oh, yes - forgot about that change; I can and will do that in a follow-up PR, sure.

Reply:
Put up a PR, here: https://github.com/mozilla/stubattribution-tests/pull/100/files